go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/dotse/epp-lib
+
+go 1.18
+
+require (
+	github.com/beevik/etree v1.1.0
+	github.com/pkg/errors v0.9.1
+	github.com/sirupsen/logrus v1.9.0
```

Comment:
Can we get rid of logrus as a dependency since we have the `Logger` interface?

Reply:
Sure, it's only used in tests right now.